e2e/wdio/core/fixtures/appData/grid-list-content.ts

```diff
@@ -0,0 +1,16 @@
+const text = 'Footer';
+const productTitle = 'Product';
+const warningColor = 'rgb(187,0,0)';
+const errorColor = 'rgb(233,115,12)';
```

Comment:
is it work for all browsers?

Reply:
agree, need to check IE and Safari now because we will unskip them eventually. For object number, I had to use (233,155,12) format without 'rbg'